client/src/App.jsx

```diff
@@ -1,9 +1,53 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+// import Counter from './Counter';
 
-const App = () => (
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: 0,
+    };
+    this.addCounter = this.addCounter.bind(this);
+    this.subtractCounter = this.subtractCounter.bind(this);
+  }
+
+  addCounter() {
+    this.setState((preveState) => ({
```

Comment:
is this an abbreviation for previous state? If so, the second e seems odd. Recommended prevState

Reply:
@rickboot Make sense. I will edit it. Thanks!